Fix typo in logging

lmagyar opened the pull request:
This is how all other FileSystemView writes out the trace info.

wolpi commented:
I like the old code more because:
* I prefer to not change passed in parameters (I could mark all of them final, nearly everywhere)
* With variable name `abs` i wanted to point out that the absolute path is used
* In log output the original parameter is logged which I find helpful to understand what is going on

In (some of) other impls both paths are logged (the passed in one and the absolute one).
E.g. in SAF (here it overrides the paramter which I often avoid):
```
        logger.trace("getFile({})", file);

        file = absolute(file);
        logger.trace("  getFile(abs: {})", file);
```

and Virtual:
```
        String absoluteVirtualPath = absolute(file);
        logger.debug("getFile '{}', absolute: '{}'", file, absoluteVirtualPath);
```

lmagyar commented:
I agree, reverted. But the old code has a bug, it logs `file` instead of `abs`, so I fixed that.

And made a separate commit (can be reverted if you don't like it), where I changed all other getFile implementations to ***not*** change passed in `file` parameter, but use a new `String abs`.

wolpi commented:
Nice to clean that up
